### DISCUSSION TITLE
Amity Online university student - unable to verify

### DISCUSSION THREAD

**Discussion by jrsarath** [community/community#132069] [GitHub Education]
### Hi everyone,

I am unable to verify my student status, i have verified with the same school before but this time its not working even tho im an active student, no matter what i do it fails

**Comment by bacngv** [comment]
Wow

**Comment by iamdulanga** [comment]
Hello, @jrsarath 

1. First try to find, the official email address that university gave to you.
2. If they did not try different but same name when choose the the institute.
3. Sometimes it takes more than 4 times of attempts to verifying the account.
4. Also use your email address that used on your github account.
5. Upload a blurless clean picture or the required document github ask for.
6. Last add correct address for the payment details on github settings (the address).

I hope this will help you a lot..!!
Dulanga...
